README.md

```diff
@@ -15,6 +15,8 @@ Building by hand
 ================
 To get up and running the absolute fastest, just use `build.sh`.  However, it builds everything and uses "reasonable" settings.  If you want something special, for now at least, run `cmake` and `make` by hand.  Better documentation is definitely welcomed, but for now, you can look at `build.sh` for reference.
 
+`build.sh` create all the DMGs.
```

Comment:
What which of the two descriptions are correct?

Reply:
I have a serious git issue on this repo.  I'm going to do a clean clone, I have literally no idea what is going on.

I don't have any branches on my machine that have both of those lines in them.